examples/basic-form/src/main.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { render, StyleSheet, View } from 'react-sketchapp';
+import { Text } from 'react-primitives';
+import { colors, typography, spacing, typeRamp, fontFamily } from './designSystem';
+import type { User } from './types';
+import DATA from './data';
+
+import Register from './components/Register';
+import Space from './components/Space';
+import TextBox from './components/TextBox/primitive';
+import Button from './components/Button';
+
+
+const styles = StyleSheet.create({
+  register: {
+    backgroundColor: colors.LightGrey,
+    padding: spacing.Large
+  },
+  heading: {
+    color: colors.Purple,
+    fontSize: typeRamp.Medium,
+    fontFamily: fontFamily,
+    fontWeight: 'bold',
+    textAlign: 'center',
+    marginBottom: spacing.Medium,
+    width: 300
+  }
+});
+
+const Page = ({ sessions }: { sessions: Array<Session> }) => (
+  <View>
+    <Text style={typography.Heading}>Basic Form w/ DOM elements and React Primitives</Text>
+    <Register>
+      {sessions.map(session => (
+        <Space key={session.password} h={spacing.Large} v={spacing.Large}>
+          <View style={styles.register}>
```

Comment:
I'd extract this whole thing to a new component ~= `<RegistrationForm>` that takes an email & password - then you could reuse it in the web version too